.manylinux-install.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+# Kill the build if anything errors
+set -e -x
+
+# Compile wheels
+for PYBIN in /opt/python/*/bin; do
+    if [[ "${PYBIN}" == *"cp27"* ]] || \
+       [[ "${PYBIN}" == *"cp33"* ]] || \
+       [[ "${PYBIN}" == *"cp34"* ]] || \
+       [[ "${PYBIN}" == *"cp35"* ]]; 
+    then
+        "${PYBIN}/pip" install -r /io/requirements-doc.txt
+        "${PYBIN}/pip" install pytest
+        "${PYBIN}/pip" install -U cython
+        "${PYBIN}/pip" install argparse
+        "${PYBIN}/cython" /io/pycrfsuite/_pycrfsuite.pyx --cplus -a -I /io/pycrfsuite
```

Comment:
Are you comfortable introducing a conditional block into `update_cpp.sh` to differentiate between normal/manylinux builds? The filepaths for the code are slightly different between the two builds, since docker has to mount all of the files to the temporary `/io/` directory in order to use them.